### ISSUE TITLE
Default plugin size

### ISSUE THREAD

**Issue by jatinchowdhury18** [ffAudio/foleys_gui_magic#39]
Hi ffAudio,

Thanks so much for this module, I've been using it on a couple projects now, and really love it!

It seems like the default plugin window size is 600 x 400, and I was wondering if there's a way to customize this? I tried changing the dimension parameters of the root view, but that didn't seem to work. What did work was hardcoding values in `MagicPluginEditor::updateSize()`:

```cpp
void MagicPluginEditor::updateSize()
{
    int width = builder->getStyleProperty (IDs::width, builder->getGuiRootNode());
    if (width < 10) width = 600;
    int height = builder->getStyleProperty (IDs::height, builder->getGuiRootNode());
    if (height < 10) height = 600; // I changed this from the default (400)
    
   // rest of function ...
}
```

Is there a better way to do this? Or if this is the best way to do it, it would be nice if the default dimensions could be set as arguments to the constructor, i.e.

```cpp
return new foleys::MagicPluginEditor (magicState, BinaryData::myGUI, BinaryData::myGUI_size, width, height);
```

Thanks,
Jatin

**Comment by ffAudio** [comment]
Thank you for the report. I just added that in the develop branch, and it should be merged to the master branch in the coming week.
You will see the width and height, as well as min and max width and height selectable when you select the root node.
Also the resizable option will be customisable.

**Comment by jatinchowdhury18** [comment]
Awesome, thank you!

**Comment by ffAudio** [comment]
In case you are curious, see here:
06f406a02636cd370b4a82d551a9de53644fb72a

**Comment by ffAudio** [comment]
This is now in the Released version 1.1.3
Click on the root node and in the top group you find the sizes. You need to close and reopen the editor for the new size constraints to take effect.
